src/gateway/router/routes/deployContractRoute.ts

```diff
@@ -1,34 +1,60 @@
 import Router from '@koa/router';
 import Transaction from 'arweave/node/lib/transaction';
 import Arweave from 'arweave';
-import {GQLTagInterface, SmartWeaveTags} from 'warp-contracts';
-import {evalType} from '../../tasks/contractsMetadata';
-import {getCachedNetworkData} from '../../tasks/networkInfoCache';
-import {BUNDLR_NODE2_URL} from '../../../constants';
-import {uploadToBundlr} from './sequencerRoute';
-import {updateCache} from "../../updateCache";
+import { GQLTagInterface, SmartWeaveTags } from 'warp-contracts';
+import { evalType } from '../../tasks/contractsMetadata';
+import { getCachedNetworkData } from '../../tasks/networkInfoCache';
+import { BUNDLR_NODE2_URL } from '../../../constants';
+import { uploadToBundlr } from './sequencerRoute';
+import { updateCache } from '../../updateCache';
+// @ts-ignore
```

Comment:
can be deleted after upgrading typescript and setting `moduleResolution` to `nodenext`, current ts version does not support this property. some other compilation errors will need to be fixed after ts upgrade so i decided to create a separate pr for this one. probably together with linter addition :))

Reply:
Please create a separate issue for this task.

Reply:
also, remember that last time you've used the linter in the gateway code - some raw sql queries were fucked up :-)